api/src/main/java/org/openmrs/validator/PersonAddressValidator.java

```diff
@@ -63,12 +63,21 @@ public void validate(Object object, Errors errors) {
 			log.debug(this.getClass().getName() + ".validate...");
 		}
 		
-		if (object == null) {
+		/*if (object == null) {
 			throw new IllegalArgumentException("The personAddress object should not be null");
-		}
+		}*/
 		
 		PersonAddress personAddress = (PersonAddress) object;
-		
+		try {
+			// Validate that the person Address  object is not null
+			if (personAddress == null) {
+				errors.reject("error.name");
+			}
+		}
+		catch (Exception e) {
+			errors.reject(e.getMessage());
+		}
+	}
 		//resolve a shorter name to display along with the error message
 		String addressString;
 		if (StringUtils.isNotBlank(personAddress.getAddress1())) {
```

Comment:
We get a compilation error:

> [ERROR] /home/travis/build/openmrs/openmrs-core/api/src/main/java/org/openmrs/validator/PersonAddressValidator.java:[85,24] error: ';' expected

Reply:
@achilep  on top of what @fruether  has pointed out  , it's good practice to run  "mvn clean package" before you push, it will save you hours of debugging before pushing. Have you looked at https://wiki.openmrs.org/display/docs/Pull+Request+Tips